tools/coverage_system_tests.rc

```diff
@@ -1,3 +1,12 @@
 [run]
-include =
-    */library.py
+omit =
+    */__init__.py
+    */_attributes.py
+    */_converters.py
```

Comment:
Do we have coverage for driver-specific _converters elsewhere?

Reply:
There are no driver specific converters. `_converters.py` is a superset of all of them that are used, and nifake has tests for all of them, plus the tests at the bottom of the file.